.github/workflows/website_cd.yml

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2022 Sharezone UG (haftungsbeschränkt)
+# Licensed under the EUPL-1.2-or-later.
+#
+# You may obtain a copy of the Licence at:
+# https://joinup.ec.europa.eu/software/page/eupl
+#
+# SPDX-License-Identifier: EUPL-1.2
+
+name: website-cd
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        environment:
+          - name: dev
+            projectId: sharezone-debug
+            serviceAccountSecret: FIREBASE_SERVICE_ACCOUNT_SHAREZONE_DEBUG
+          - name: prod
+            projectId: sharezone-c2bd8
+            serviceAccountSecret: FIREBASE_HOSTING_PROD_KEY
+    steps:
```

Comment:
Wait, why a matrix with both?

Reply:
For production (https://sharezone.net) and dev (https://dev.sharezone.net). In dev we can test our redirects and in the next weeks we will also have prod/dev in the website Flutter code (because of the checkout sesion and prod/dev Stripe integration).

Reply:
<img width="710" alt="image" src="https://github.com/SharezoneApp/sharezone-app/assets/24459435/3c69f4ae-a65e-4781-b0b5-36eafe7c637b">

Two values, not one